### DISCUSSION TITLE
Assign Snap Areas

### DISCUSSION THREAD

**Discussion by emmtre** [rxhanson/RectanglePro-Community#323] [General]
Any special reason why you can't assign the snap to corner shortcuts (top left, top right, bottom left and bottom right) to the corners in the snap areas?

![screenshot](https://github.com/rxhanson/RectanglePro-Community/assets/14124095/dc7305f1-5aa8-4d4c-87f4-c5b4624451db)

**Comment by rxhanson** [comment]
I think I'm missing something here.

![image](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/5ee1979b-2750-479b-be00-85cdef69b114)

The corners there aren't in the dropdown menu for you?

**Comment by emmtre** [comment]
I'm missing **_Snap_** Top Left, **_Snap_** Top Right, **_Snap_** Bottom Left and **_Snap_** Bottom Right as choices in Snap Areas. The ones in your screenshot are **_Corners_** Top Left, **_Corners_** Top Right, **_Corners_** Bottom Left and **_Corners_** Bottom Right. Sorry if I wasn't clear enough in my first post. You can choose fill, resize and snap, stash, etc in the drop down menus but not just snap to corner.

**Reply by rxhanson** [reply]
Got it, thanks for the clarification. The reason those are not included is mainly just to keep the list a little slimmer. I had made the assumption that those wouldn't be necessary, as most users would likely just drag a window to the corner and leave it, rather than dragging the cursor to the corner of the display while dragging a window. I'll consider adding this in next time I'm in that part of the code.

**Comment by emmtre** [comment]
Many thanks for considering adding snap to corner as well. At least for me it makes sense when using a small MacBook screen. I'm using too many apps that don't remember the window position. But I guess it all depends on personal preference, screen size, using mouse or trackpad, etc. Keep up the good work.

**Comment by emmtre** [comment]
Is there any chance this requested feature will be added in any upcoming version? I really miss this feature daily when I work on a smaller MacBook Pro screen.
